src/main/java/org/recommendation/model/BaseModel.java

```diff
@@ -0,0 +1,14 @@
+package org.recommendation.model;
+
+public class BaseModel {
+    private String userId;
```

Comment:
we have have only id instead of userId so that we can use this baseModel in different model

Reply:
done